examples/cpp/inference/infer_single_network.cc

```diff
@@ -17,7 +17,6 @@
 #include <thread>
 
 #include "tensorflow/cc/client/client_session.h"
```

Comment:
Not sure if this is tested as part of the CI, so might need to be tested locally to make sure it works with 2.2

Reply:
It works.